app/src/main/res/layout/fragment_home.xml

```diff
@@ -140,16 +140,23 @@
                 android:layout_height="24dp"
                 android:layout_gravity="start|center_vertical"
                 android:layout_marginStart="8dp"
+                android:layout_marginEnd="12dp"
                 android:clickable="false"
                 android:focusable="false"
-                android:importantForAccessibility="no" />
+                android:importantForAccessibility="no"/>
+
+            <org.mozilla.fenix.search.toolbar.SearchSelector
```

Comment:
Could we set the marginTop here?

Reply:
It won't affect the top margin of the material card inside the `[search_selector](https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/layout/search_selector.xml#15)`